generic-server/src/plugin/container.rs

```diff
@@ -89,6 +88,35 @@ impl<'a> PluginContainer<'a> {
         }
     }
 
+    pub fn unload(&mut self) -> Result<(), PluginContainerError> {
+        // Unmount plugins and clearing the vector of routes
+        let errors: HashMap<_, _> = self
+            .plugins
+            .iter()
+            .filter_map(|plugins| match plugins.unmount() {
+                Ok(_) => {
+                    tracing::info!("Unmounted plugins {}", plugins.name());
```

Comment:
Be consistent with the rest of the codebase, start the log message with a lowercase letter.
```suggestion
                    tracing::info!("unmounted plugin {}", plugin.name());
```